src/lib/addons/addon.test.ts

```diff
@@ -16,6 +16,7 @@ const ARGS: IAddonConfig = {
     unleashUrl: url,
     integrationEventsService: {} as IntegrationEventsService,
     flagResolver: {} as IFlagResolver,
+    eventBus: {} as any,
```

Comment:
Maybe it's okay because it's a test, idk, but I hate `any`. I see this file `src/test/e2e/features-created-by-user-id-data-migration.e2e.test.ts` just seems to import the event bus without problems. Maybe we can do that?

Reply:
thanks, I'll have a look at that!